readthedocs/projects/migrations/0024_add-show-version-warning.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2018-05-02 01:27
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import readthedocs.core.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0023_migrate-alias-slug'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='show_version_warning',
+            field=models.BooleanField(default=False, help_text='Show warning banner in non-stable versions.', verbose_name='Show version warning'),
```

Comment:
> in non-stable versions

I think this should be different since the banner isn't shown in the latest version also

Reply:
Yes, agree. Please, fix this in the model and here.

Reply:
Any suggestions? Maybe Show warning banner in old versions?

Reply:
I liked the "non-stable" since it's clear. Maybe:

> Show warning banner in non-stable nor latest versions

What do you think?

Reply:
I wanna to avoid the `nor`, but it's better than old versions